code/datums/components/material_container.dm

```diff
@@ -69,6 +69,9 @@
 		return
 	if(I.flags & ABSTRACT)
 		return
+	var/obj/machinery/machine_parent = parent
+	if(machine_parent.panel_open)
+		return
```

Comment:
Is this here because using a multitool with the panel opens whacks the exofab?

Reply:
Yes, it tries to ingest it every time.